integration/testdata/fixtures/fs/yarn/node_modules/jquery/package.json

```diff
@@ -0,0 +1,118 @@
+{
+    "name": "jquery",
+    "title": "jQuery",
+    "description": "JavaScript library for DOM operations",
+    "version": "3.2.1",
+    "main": "dist/jquery.js",
+    "homepage": "https://jquery.com",
+    "author": {
+      "name": "OpenJS Foundation and other contributors",
+      "url": "https://github.com/jquery/jquery/blob/3.7.0/AUTHORS.txt"
+    },
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/jquery/jquery.git"
+    },
+    "keywords": [
+      "jquery",
+      "javascript",
+      "browser",
+      "library"
+    ],
+    "bugs": {
+      "url": "https://github.com/jquery/jquery/issues"
+    },
+    "license": "MIT",
```

Comment:
Your golden file doesn't contain license.

Reply:
Good trick! I forgot to move this file after renaming fs to repo.

Reply:
done